public/assets/sass/style.scss

```diff
@@ -125,6 +125,13 @@ md-toolbar {
 
 .ngdialog {
   overflow: hidden;
+  .tree-selected {
```

Comment:
please fix. it looks like you can actually make this one css selector using `.tree-selected file-name {}`